contracts/BalanceTrackerBase.sol

```diff
@@ -73,29 +75,29 @@ abstract contract BalanceTrackerBase {
         buyBackBurner = _buyBackBurner;
     }
 
-    /// @dev Adjusts initial requester balance accounting for max request delivery rate (debit).
+    /// @dev Adjusts initial requester balance accounting for delivery rate (debit).
     /// @param balance Initial requester balance.
-    /// @param maxDeliveryRate Max delivery rate.
```

Comment:
This function is used for any arbitrary delivery rate, not just a single max one.